docs/playbook-reference/triggers/prometheus.rst

```diff
@@ -29,7 +29,9 @@ The following triggers are available for Prometheus alerts:
             customPlaybooks:
             - triggers:
               - on_prometheus_alert:
-                  alert_name: HostHighCpuLoad
+                  scope:
```

Comment:
I think this example is not correct. 
I think the scope, supports only `labels` and `annotations` (and maybe `status`)
So maybe the example should be:
```
            customPlaybooks:
            - triggers:
              - on_prometheus_alert:
                  alert_name: HostHighCpuLoad
                  scope:
                    include::
                    - labels:
                       - "deployment=nginx"
```

I think it's important to explain the scope only support `labels` and `annotations`